test/Modules/runtests.jl

```diff
@@ -10,3 +10,10 @@ include("local_rings.jl")
 include("MPolyQuo.jl")
 include("homological-algebra.jl")
 include("ProjectiveModules.jl")
+
+# FIXME: broken, probably obsolete?
+# include("GradedModules.jl")
+
+
+# TODO: the corresponding source file is also commented
+# include("FreeModules-graded.jl")
```

Comment:
Theses tests were disabled in 955b60dbcc13c34b65f1660cdad2a3e5a1aadaef

I think they are for the "old" graded modules as implemented by @fieker... Dunno if we should keep them, or just remove them... @fieker ?